examples/ProducerConsumerInstrument/ProducerConsumerInstrument.go

```diff
@@ -0,0 +1,156 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/DARA-Project/GoDist-Scheduler/propchecker"
+	"log"
+	"os"
+	"runtime"
+	"runtime/pprof"
+)
+
+type Consumer struct {
+	msgs *chan int
+}
+
+// NewConsumer creates a Consumer
+func NewConsumer(msgs *chan int) *Consumer {
+	runtime.ReportBlockCoverage("../examples/ProducerConsumerInstrument/ProducerConsumerInstrument.go:18:21")
+	runtime.ReportBlockCoverage("../examples/ProducerConsumerInstrument/ProducerConsumerInstrument.go:18:103")
```

Comment:
It seems like the instrumentation was run multiple times? Can you confirm that you ran Type A instrumentation only once?